### PR TITLE
+ create HomePage

### PR DESCRIPTION
**Pull request by MeteorPro**
Add home page, header with name and icon, find field, categories

### CODE REVIEW

File: src/components/Categories/Categories.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  .categories-scroll {
+    display: flex;
+  }
+  
+  .category {
+    width: 100px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: flex-start;
```
Comment (TaniaMitnovych) [inline src/components/Categories/Categories.scss:15]: @include flex(column, flex-start, center);

File: src/components/Categories/Categories.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    margin: 20px 10px 20px;
+    position: relative;
+  }
+  
+  .category-image {
+    background-color: white;
```
Comment (TaniaMitnovych) [inline src/components/Categories/Categories.scss:21]: background-color: $white;

File: src/components/Categories/Categories.scss [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    box-shadow: 1px 1px 2px rgba(0, 0, 0, 0.2);
+    border-radius: 10px;
+    outline: none;
+    padding: 10px;
+    width: 70px;
+    height: 70px;
```
Comment (TaniaMitnovych) [inline src/components/Categories/Categories.scss:27]: @include set-size(70px,70px);

File: src/components/HomePageHeader/HomePageHeader.scss [hunk 4]
```diff
@@ -0,0 +1,36 @@
+.header{
+    display: flex;
+    align-items: center;
+    padding: 1%;
+    justify-content: space-between;
```
Comment (TaniaMitnovych) [inline src/components/HomePageHeader/HomePageHeader.scss:5]: @include flex($items:center, $content:space-between);

File: src/components/HomePageHeader/HomePageHeader.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './HomePageHeader.scss';
+import useUser from '../../hooks/useUser';
+import { useAuth } from '../../hooks/useAuth';
+import defaultUserImage from '../../assets/images/user-icon.png'
+
+export const HomePageHeader: React.FC<{}> = ({}) => {
+    const { isAuthenticated } = useAuth();
+    const user = useUser();
```
Comment (TaniaMitnovych) [inline src/components/HomePageHeader/HomePageHeader.tsx:9]:
```
const { user } = useUser();

File: src/components/HomePageHeader/HomePageHeader.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+    const greeting = !isAuthenticated ? `Welcome, ${user.name} 👋` : 'Welcome, guest 👋';
+    const UserImage = !isAuthenticated ? user.image : defaultUserImage ;
```
Comment (TaniaMitnovych) [inline src/components/HomePageHeader/HomePageHeader.tsx:12]:
```
const UserImage = isAuthenticated ? user.image : defaultUserImage ;
```
Same to prev

File: src/routes/Routes.tsx [hunk 7]
```diff
@@ -1,10 +1,13 @@
 import {Route, Switch} from 'react-router-dom';
-import {MainPage} from '../pages';
+import {MainPage, HomePage} from '../pages';
```
Comment (TaniaMitnovych) [inline src/routes/Routes.tsx:2]:
```
import { MainPage, HomePage } from '../pages';
```
Add spaces inside the braces

File: src/components/HomePageHeader/HomePageHeader.tsx [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+import './HomePageHeader.scss';
+import useUser from '../../hooks/useUser';
+import { useAuth } from '../../hooks/useAuth';
+import defaultUserImage from '../../assets/images/user-icon.png'
+
+export const HomePageHeader: React.FC<{}> = ({}) => {
+    const { isAuthenticated } = useAuth();
+    const user = useUser();
+
+    const greeting = !isAuthenticated ? `Welcome, ${user.name} 👋` : 'Welcome, guest 👋';
```
[the remark this answers is not in the document]
Comment (TaniaMitnovych) [reply src/components/HomePageHeader/HomePageHeader.tsx]: isAuthenticated must be 'false' by default (this bug must be fixed later), so you should act basing on this info

**Review by TaniaMitnovych — APPROVED** [review]

**Review by NazarMykhailiv — APPROVED** [review]

**Pull request merged**